recipes-bsp/rpi-boot-setting/files/config.txt

```diff
@@ -0,0 +1,15 @@
+[all]
+boot_delay=1
+kernel=u-boot.bin
+
+# Put the RPi3 into 64 bit mode
+arm_control=0x200
```

Comment:
We need to use arm_64bit instead of  arm_control because it is [deprecated](https://www.raspberrypi.com/documentation/computers/config_txt.html#arm_control).

Reply:
Fixed here.

https://github.com/miraclelinux/meta-emlinux/compare/9b2e7965f9aff46776bbcd0ea47e5c976e421b0d..b42989cea99efb1d6e43592daa3873e1a25b8a7b

Reply:
thanks